### DISCUSSION TITLE
Streamline chemistry production process with UI elements

### DISCUSSION THREAD

**Discussion by tron359** [unitystation/unitystation#8419] [Ideas]
Making chemistry products takes a long time between reading the wiki and clicking in-game. For players currently wearing a radio or UI device, a selectable or intelligent-populating list of chemicals for production might be shown with the next step highlighted or otherwise colored.

Alternatively, the ability to "save" a complex chemical, either via a memory disk or other tool, would be lovely. Especially as the more complicated recipes, especially when mass-produced, can take up to 20 minutes between reading the wiki and clicking in-game.

Department heads might also be given the ability to send prioritized work orders to staff, showing up as marked boxes within the UI element. Perhaps even giving the staff the ability to mark them correct, whether or not they've actually done so ;) 

Once designed and implemented, this same system could be added to any of the other departments, especially botany, security, and the kitchen (ticket orders, anyone?).

**Comment by braincrab** [comment]
perhaps simple recepies should be displayed at the start (ie:ammonia, oil) and more complex ones (ie:dithalamine) become displayed as its upgraded? with guides for dangerous scheme appearing if its emagged.
